lib/access/oai.rb

```diff
@@ -113,6 +113,8 @@ def dcFields(xml)
     # Second version of identifier to contain an item link. I (MH) thought this was just for OCLC, but am
     # informed by folks at oaDOI that it's pretty common and preferred for all repos.
     xml.tag!("dc:identifier", "https://escholarship.org/uc/item/#{@data['id'].sub(%r{^ark:/13030/qt}, '')}")
+    xml.tag!("dc:identifier", @data['contentLink'])
```

Comment:
Is the content link a link to eschol or something else?

Reply:
yes, it's the eschol contentLink.  It's the same value as the[ contentLink field in the API.](https://escholarship.org/graphql/iql?query=%7B%0A%20%20items%7B%0A%20%20%20%20nodes%7B%0A%20%20%20%20%20%20id%0A%20%20%20%20%20%20contentLink%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D)